settings/chains/prod/ink.yaml

```diff
@@ -0,0 +1,16 @@
+network_name: ink
+chain_id: 57073
+rollup_type: op_stack
+dao:
+  crv: "0x0"
+  crvusd: "0x0"
+explorer_base_url: https://explorer.inkonchain.com/
+
+# Not related to development, for further integrations
+layer: 2
```

Comment:
could you please add additional information here. see example: https://github.com/curvefi/curve-core/blob/main/settings/chains/devnet/hyperliquid.yaml